### DISCUSSION TITLE
Future release dates

### DISCUSSION THREAD

**Discussion by amanverma-ism** [cefsharp/CefSharp#4520] [General]
Where can we see the future release dates in general? Right now, I want to know when we will get the nuget package for CEF 114?

**Comment by amaitland** [comment]
You can subscribe to [Release Only Notifications](https://github.blog/changelog/2018-11-27-watch-releases/) here on `GitHub` to be notified of releases. Version `114` was released a few days ago.

We rely on [Spotify Builds](https://cef-builds.spotifycdn.com/index.html) for official releases. Spotify/CEF doesn't provide a calendar, so it's not practical to provide one for this project.
